src/app/app.component.ts

```diff
@@ -357,6 +364,13 @@ export class CopayApp {
       this.platformProvider.isCordova &&
       this.appProvider.info.name === 'bitpay'
     ) {
+      if (cards) {
+        this.events.publish('CardAdvertisementUpdate', {
```

Comment:
Be careful sending an event here. This is the first component the app will load before tabs.ts and home.ts. So, if subscription to this event is in home.ts, it won't be called (I guess)